Message BoardUITests/Message_BoardUITests.swift

```diff
@@ -23,4 +23,87 @@ class Message_BoardUITests: XCTestCase {
         app.launch()
     }
     
+    func testNewMessage() {
+        firstMessageThreadCell.tap()
+        addMessageButton.tap()
+        
+        nameTextField.tap()
+        nameTextField.typeText("Sean1")
+        
+        messageTextView.tap()
+        messageTextView.typeText("Test1")
+        
+        XCTAssertTrue(sendMessageButton.exists)
+        sendMessageButton.tap()
+        
+        XCTAssertTrue(firstMessageThreadDetailCell.isHittable)
+    }
+    
+    func testMessageThreadCellsContainTitles() {
+        XCTAssertTrue(messageThreadTableView.cells.staticTexts["A New Thread"].exists)
+        XCTAssertTrue(messageThreadTableView.cells.staticTexts["Testing again"].exists)
+    }
+    
+    func testMessageThreadDetailCellContainsTitleAndSubtitle() {
+        firstMessageThreadCell.tap()
+        XCTAssertFalse(messageThreadDetailTableView.cells.staticTexts["Title"].exists)
+        XCTAssertFalse(messageThreadDetailTableView.cells.staticTexts["Subtitle"].exists)
+    }
+    
+    var sendMessageButton: XCUIElement {
```

Comment:
Great work organizing buttons so they would be easy to use.

It might make sense to think about organize these elements into "Pages" so that you don't have two "add" buttons when trying to write your UI tests. 

I'd like to see you extra duplicated code `let app = XCUIApplication()` ... granted the instructions said not to modify setUp(), it should have been more clear the flag was important.